### ISSUE TITLE
new tag creation - option:beforeCreated

### ISSUE THREAD

**Issue by fabiodp1** [sagalbot/vue-select#1748]
Hi, I couldn't find a way "out of the box" to manage new tag creation and validation before it is created, for example a "beforeCreated" event.
For instance I'd need to check if the new tag contains non-alphanumeric tags, and if so show a warning and stop the tag creation, stopping the push, state update and consequent events.
Thanks

**Comment by gwenf** [comment]
Yes, this is what I think I need as well. I need some way to enforce a character limit on new options.
